eng/pipelines/templates/variables/globals.yml

```diff
@@ -17,3 +17,7 @@ variables:
   DOTNET_NOLOGO: 1
   Package.EnableSBOMSigning: true
   Codeql.SkipTaskAutoInjection: true
+  # Define optional AdditionalTestFilters via pipeline runtime variables or matrix config variables
+  # Variable value must start with either & or | to join conditions.
```

Comment:
The comment about starting with & can be removed now.

Reply:
Whoops, I updated the wiki doc but missed this one.